PWGHF/vertexingHF/AliAnalysisTaskSEHFSystPID.h

```diff
@@ -196,6 +213,18 @@ class AliAnalysisTaskSEHFSystPID : public AliAnalysisTaskSE {
   float fCutGeoNcrNclGeom1Pt;                                                        /// 3rd parameter geometrical cut
   float fCutGeoNcrNclFractionNcr;                                                    /// 4th parameter geometrical cut
   float fCutGeoNcrNclFractionNcl;                                                    /// 5th parameter geometrical cut
+  
+  float fCutMinCascRadius;                                                           /// min radius of the cascade
+  float fCutMinV0Radius;                                                             /// min radius of the V0
+  float fCutMaxV0Radius;                                                             /// max radius of the V0
+  double fCutCosPA;                                                                  /// cosinus of the pointing angle
+  float fCutDcaBachToPV;                                                             /// DCA between bachelor and PV
+  float fCutDcaV0ToPV;                                                               /// DCA between V0 and PV
+  float fCutDcaV0DaughToPV;                                                          /// DCA between V0 daughter and PV
+  float fDcaV0Daught;                                                                /// DCA between the V0 daughters
+  float fDcaCascDaught;                                                              /// DCA between the cascade daughters
+  float fCutInvMassLam;                                                              /// Width of the lambda
+  float fCutNSigmaPID;                                                               /// N of sigmas for PID
```

Comment:
the ClassDef should also be incremented since you are adding new persistent members

Reply:
yes, I have incremented it